website/src/app/views/about/members.js

```diff
@@ -59,6 +59,8 @@ import ted from '../../../images/about/memberImages/ted.jpg';
 import phil from '../../../images/about/memberImages/phil.jpg';
 import kory from '../../../images/about/memberImages/kory.jpg';
 import charles from '../../../images/about/memberImages/charles.jpg';
+import keagan from '../../../images/about/memberImages/keagan.jpg';
+
```

Comment:
Can you remove this blank line pls :) Then should be good to go.

Reply:
Got it :)